### PR TITLE
fix(ios): implement statusTap for iOS 13

### LINKED ISSUE #2109: statusTap event not working on iOS 13
# Bug Report

## Capacitor Version

`npx cap doctor` output:
```
💊   Capacitor Doctor  💊

Latest Dependencies:

  @capacitor/cli: 1.3.0

  @capacitor/core: 1.3.0

  @capacitor/android: 1.3.0

  @capacitor/ios: 1.3.0

Installed Dependencies:

  @capacitor/cli 1.3.0

  @capacitor/core 1.3.0

  @capacitor/android 1.3.0

  @capacitor/ios 1.3.0


[success] Android looking great! 👌
  Found 4 Capacitor plugins for ios:
    @biesbjerg/capacitor-music-player (0.0.1)
    @teamhive/capacitor-email (2.1.2)
    capacitor-analytics (0.0.4)
    cordova-plugin-inapppurchase (1.2.0)
[success] iOS looking great! 👌
```

## Affected Platform(s)

- [ ] Android
- [x] iOS
- [ ] Electron
- [ ] Web

## Current Behavior

When tapping on status bar ion-content does not scroll to top

## Expected Behavior

Scroll to top on tap

### PR DESCRIPTION
**Pull request by jcesarmobile**
use swizzling to implement statusTap for iOS 13

closes #2109

### CODE REVIEW

File: ios/Capacitor/Capacitor/UIStatusBarManager+CAPHandleTapAction.m [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                          swizzledSelector,
+                          method_getImplementation(originalMethod),
+                          method_getTypeEncoding(originalMethod));
+    } else {
+      method_exchangeImplementations(originalMethod, swizzledMethod);
+    }
+  });
+}
+
+-(void)nofity_handleTapAction:(id)arg1 {
```
Comment (Proper-Job) [inline ios/Capacitor/Capacitor/UIStatusBarManager+CAPHandleTapAction.m]: Is this a typo or intentionally spelled wrong?

Comment (jcesarmobile) [reply ios/Capacitor/Capacitor/UIStatusBarManager+CAPHandleTapAction.m]: typo

**Comment by Proper-Job** [comment]
Isn't handleTapAction: a private API?
Don't we run a risk of Apple rejecting capacitor based apps because of this?

**Comment by jcesarmobile** [comment]
We are technically not using it, just listening for its invocation. We have used similar approaches in different Cordova plugins and Capacitor over the years to workaround other iOS bugs without problems.

But if Apple ever rejects or warns about it, we would remove it.

**Pull request merged**